backend/app/app.go

```diff
@@ -121,5 +125,19 @@ func Start(
 		panic(err)
 	}
 
-	httpAPI.StartAndWait(config.HTTPAPIPort)
+	httpAPI.StartAsync(config.HTTPAPIPort)
+
+	gRPCAPI := dep.InjectGRPCApi(sqlDB)
+	gRPCService, err := service.
+		NewGRPCBuilder(config.LogPrefix).
+		RegisterHandler(func(server *grpc.Server) {
+			gRPCAPI.RegisterServers(server)
+		}).
+		Build()
+
+	if err != nil {
+		panic(err)
+	}
```

Comment:
We should consider using the DI to inject logger. If not, everything will be logged to Datadog during local development. That's slow and inconvenient.